tests/benchmarks/android_geekbench.py

```diff
@@ -64,19 +64,17 @@ class GeekbenchTest(LisaBenchmark):
     bm_collect = 'ftrace energy'
 
     def benchmarkInit(self):
+        super(GeekbenchTest, self).benchmarkInit()
         self.setupWorkload()
         self.setupGovernor()
-        if self.reboot:
-            self.reboot_target()
 
     def benchmarkFinalize(self):
         if self.delay_after_s:
             self._log.info("Waiting %d[s] before to continue...",
                            self.delay_after_s)
             sleep(self.delay_after_s)
 
-    def __init__(self, governor, test, reboot=False, delay_after_s=0):
-        self.reboot = reboot
```

Comment:
In this implementation it was technically possible to reboot only on some instances... with the new API (which uses command line params) I think we miss that.

Well... maybe we can re-add it on-demand the first time we find it useful ;-)